docs/_posts/2020-05-25-jekyll-4-1-0-released.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Jekyll 4.1.0 Released'
+date: 2020-05-25 19:25:30 +0530
+author: ashmaroli
+version: 4.1.0
+category: release
+---
+
+Hello Jekyllers!
+
+It's time for yet another release that includes enhancements, optimizations and bug-fixes. Highlights of this release
+are:
+
+* Jekyll now supports rendering excerpts for *pages* in addition to documents and posts.
+* The `where_exp` filter got enhanced.  Earlier, one could just use either `and` or `or` once per expression. Now, one
+may use those binary operators multiple times in the filter's expression.
+* Jekyll has a new set of filters based on *its flavor* of the `where` and `where_exp` filters. Named `find` and
+`find_exp` filters respectively, they work similar to their ancestors except that they return **the first object** that
+satisfies the given conditions.
```

Comment:
Can we link to examples in the docs for the usage of the filters?

Reply:
Filters are documented in a single table. Individual filters don't have an anchor to point to.
The link to filter docs page is already linked in the next line (ln#20) of this post